recipes/impit/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "impit" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/impit-{{ version }}.tar.gz
+  sha256: 09ce214caf91b2bede23babc9101ab2277623ab1c9cabe4c117ce3eb012e8b38
+
+build:
+  script_env:
+    - CONDA_BUILD_SYSROOT=/opt/MacOSX14.5.sdk  # [osx]
+    - SDKROOT=$CONDA_BUILD_SYSROOT  # [osx]
+    - MACOSX_SDK_VERSION=14.5  # [osx]
+    - RUSTFLAGS=--cfg reqwest_unstable
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

I would suggest to do this as:

```suggestion
  script: env RUSTFLAGS='--cfg reqwest_unstable' {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```